<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gomoku</class>
 <widget class="QMainWindow" name="Gomoku">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gomoku</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GameBoard" name="gameboard" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>571</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>132</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.Helvetica Neue DeskInterface&quot;;
background-color: rgb(84, 105, 255);</string>
    </property>
    <property name="text">
     <string>black chess</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>85</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>connnect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="undoButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>undo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>quit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="informButton">
       <property name="text">
        <string>inform</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GameBoard</class>
   <extends>QWidget</extends>
   <header>gameboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
